tests/test_pytest/test_plugin_schema/bad/product_defaults/interpolator_algorithm_colormap.yaml

```diff
@@ -0,0 +1,53 @@
+---
+interface: product_defaults
+family: bad_family
+name: good_test
+docstring: "A product_defaults instance with a bad family"
+spec:
+  interpolator:
+    plugin:
+      name: pyresample.nearest_neighbor
+      arguments: {}
+  algorithm:
+    plugin:
+      name: single_channel
+      arguments: {}
+  colormap:
+    plugin:
+      name: matplotlib.colormap
+      arguments: {}
+
+---
+interface: product_defaults
+family: interpolator_algorithm_colormap
+name: good_test
+docstring: "A product_defaults instance missing its spec"
+
+---
+interface: product_defaults
+family: interpolator_algorithm_colormap
+name: good_test
+docstring: "A product_defaults instance missing its interpolator property"
+spec:
+  algorithm:
+    plugin:
+      name: single_channel
+      arguments: {}
+  colormap:
+    plugin:
+      name: matplotlib.colormap
+      arguments: {}
+
+---
+interface: product_defaults
```

Comment:
Are these 2 tests doing the same thing?  Missing interpolator?  Did you mean to test missing interpolator, algorithm, and colormap separately?